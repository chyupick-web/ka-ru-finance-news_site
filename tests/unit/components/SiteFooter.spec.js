import { shallowMount, createLocalVue } from '@vue/test-utils'
import Component from '@/components/SiteFooter.vue'
import BootstrapVue from 'bootstrap-vue'

const localVue = createLocalVue()
localVue.use(BootstrapVue)

describe('SiteFooter.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Component, { localVue })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
