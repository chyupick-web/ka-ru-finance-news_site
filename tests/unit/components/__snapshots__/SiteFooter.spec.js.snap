// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteFooter.vue matches snapshot 1`] = `
<b-container-stub tag="div" fluid="true" class="bg-sei-gray-darkest text-sei-gray-lightest pt-5 pb-5">
  <b-row-stub tag="div">
    <b-col-stub tag="div">
      <b-container-stub tag="div">
        <b-row-stub tag="div" class="mb-4">
          <b-col-stub md="true" tag="div" class="mb-2 mb-md-0">
            <address>
              Carnegie Mellon University<br>
              Software Engineering Institute<br>
              4500 Fifth Avenue<br>
              Pittsburgh, PA 15213-2612<br>
              <b-link-stub href="tel:412-268-5800" target="_self" event="click" routertag="a">412-268-5800</b-link-stub>
            </address>
          </b-col-stub>
          <b-col-stub md="true" tag="div" class="mb-2 mb-md-0">
            <ul class="list-social list-inline">
              <li class="list-inline-item">
                <b-link-stub href="https://www.facebook.com/SEICMU/" target="_blank" event="click" routertag="a" class="social facebook"><span class="sr-only">Facebook</span></b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://twitter.com/sei_cmu" target="_blank" event="click" routertag="a" class="social twitter"><span class="sr-only">Twitter</span></b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.linkedin.com/company/software-engineering-institute" target="_blank" event="click" routertag="a" class="social linkedin"><span class="sr-only">LinkedIn</span></b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.youtube.com/user/TheSEICMU" target="_blank" event="click" routertag="a" class="social youtube"><span class="sr-only">YouTube</span></b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://podcasts.apple.com/us/podcast/software-engineering-institute-sei-podcast-series/id566573552?mt=2" target="_blank" event="click" routertag="a" class="social itunes"><span class="sr-only">iTunes</span></b-link-stub>
              </li>
            </ul>
          </b-col-stub>
          <b-col-stub md="true" tag="div" class="mb-4 mb-md-0">
            <b-button-stub block="true" size="lg" variant="outline-light" type="button" tag="button" href="https://sei.cmu.edu/contact-us/" target="_self" class="btn-contact rounded-0 pt-4 pb-4">Contact Us</b-button-stub>
          </b-col-stub>
        </b-row-stub>
        <b-row-stub tag="div">
          <b-col-stub tag="div">
            <ul class="list-inline small">
              <li class="list-inline-item">
                <b-link-stub href="https://www.sei.cmu.edu/locations/index.cfm" target="_self" event="click" routertag="a">Office Locations</b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.sei.cmu.edu/additional-sites-directory/index.cfm" target="_self" event="click" routertag="a">Additional Sites Directory</b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.sei.cmu.edu/legal/index.cfm" target="_self" event="click" routertag="a">Legal</b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.sei.cmu.edu/legal/privacy-notice/index.cfm" target="_self" event="click" routertag="a">Privacy Notice</b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.cmu.edu/hr/ethics-hotline/" target="_self" event="click" routertag="a">CMU Ethics Hotline</b-link-stub>
              </li>
              <li class="list-inline-item">
                <b-link-stub href="https://www.sei.cmu.edu/index.cfm" target="_self" event="click" routertag="a">www.sei.cmu.edu</b-link-stub>
              </li>
            </ul>
          </b-col-stub>
        </b-row-stub>
        <b-row-stub tag="div">
          <b-col-stub tag="div"><small>Â©2020 Carnegie Mellon University</small></b-col-stub>
        </b-row-stub>
      </b-container-stub>
    </b-col-stub>
  </b-row-stub>
</b-container-stub>
`;
